main.go

```diff
@@ -958,7 +958,7 @@ func buildAndRun(pkgName string, config *compileopts.Config, stdout io.Writer, c
 	err = run(cmd, result)
 	if err != nil {
 		if ctx != nil && ctx.Err() == context.DeadlineExceeded {
-			stdout.Write([]byte(fmt.Sprintf("--- timeout of %s exceeded, terminating...\n", timeout)))
+			fmt.Fprintf(stdout, "--- timeout of %s exceeded, terminating...\n", timeout)
```

Comment:
I mean... why isn't that:
```go
fmt.Printf("--- timeout of %s exceeded, terminating...\n", timeout)
```
?

Reply:
Because `stdout` in this case is only *sometimes* `os.Stdout`, and other times we wrap it in our own buffering code.

Reply:
Figured. Thanks!